staging/src/k8s.io/apiserver/pkg/storage/etcd3/store.go

```diff
@@ -59,6 +60,9 @@ func (d authenticatedDataString) AuthenticatedData() []byte {
 
 var _ value.Context = authenticatedDataString("")
 
+// LeaseReuseLength defines the period of time in seconds that each lease is reused.
+var LeaseReuseLength int64 = 60
```

Comment:
comment on why this is a var not a const (for example, we should tell people this is for testing purpose etc..)

Reply:
done